pkgs/development/libraries/reactphysics3d/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, stdenv, fetchFromGitHub, cmake }:
+
+stdenv.mkDerivation rec {
```

Comment:
> Other than that, lgtm if it builds/works, I'll add an approval once this is fixed.

builds, tested using nixpkgs-review

Reply:
since you switched to the `finalAttrs` idiom, rather than using a let-expression, I'd suggest just using `rev = "v${finalAttrs.version}";`

Reply:
> since you switched to the `finalAttrs` idiom, rather than using a let-expression, I'd suggest just using `rev = "v${finalAttrs.version}";`

Will do, asked in discord didnt get an answer so i just put it in here to see what people would say!